msrest/paging.py

```diff
@@ -44,21 +44,17 @@ class Paged(collections.Iterable):
     def __init__(self, command, classes, raw_headers=None):
         self.next_link = ""
         self.current_page = []
+        self._current_page_iter_index = 0
         self._derserializer = Deserializer(classes)
```

Comment:
Could you take this opputunity to fix the typo in the "deRserializer" attribute? :)